.vscode/extensions.json

```diff
@@ -0,0 +1,12 @@
+{
+    "recommendations": [
+        "dart-code.dart-code",
+        "dart-code.flutter",
+        "usernamehw.errorlens",
+        "felixangelov.mason",
+        "felixangelov.bloc",
+        "nivisi.dart-build-runner-tools",
+        "tenraneko.pubspec-dependency-opener",
+        "jeroen-meijer.pubspec-assist",
+    ]
+}
```

Comment:
Enter a new line at the end

Reply:
@utpal-barman why? My formatter automatically removes it

Reply:
I guess it's POSIX where it detects each line when it ends with a new line character, GitHub has this feature enabled by default and suggests ending a new line on PR, Even Dart-formatter adds EOF by default. I don't like that red sign warning TBH 😄 

(Not sure why your IDE removes it, for me JSON file doesn't get formatted on a fresh IDE, maybe there are some plugins working on your side, if you want this feature by default you can install Prettier)

<img width="224" alt="Screenshot 2024-05-14 at 3 59 01 PM" src="https://github.com/ml-opensource/flutter-template/assets/16848599/9b8b4e63-fd61-4970-b04d-5edf16a80a86">

Reply:
Done boss. Had to run the command explicitly though  😄